scripts/common.py

```diff
@@ -128,6 +132,11 @@
 FIREBASE_EMULATOR_CACHE_DIR = (
     os.path.join(CURR_DIR, os.pardir, 'firebase_emulator_cache'))
 
+sys.path.insert(0, PY_GITHUB_PATH)
+MYPY = False
+if MYPY:  # pragma: no cover
+    import github
```

Comment:
Why do we make this conditional on the MYPY constant, which is always False?

Reply:
In the `common.py` file, we are using the `github` module for only adding type annotations, and for that, we are importing this module like other ones(within if condition). does this answer your question?

Reply:
Sorry, I'm still confused. What's the point in having the `if` condition when the condition is always `False`? Doesn't that mean that the import never happens?

Reply:
Yeah, partially true because the import never happens at `runtime`. We are setting the `MYPY` variable to False and by using the if condition, we check the value of this variable. For being `False`, it will be inert during runtime but will import the specified modules only during type checking. Here in this file, we are only using the `github` module in type annotations, so we don't need this at runtime. does that answer your question?

Reply:
Yes, it does. Since this relies on some counter-intuitive behavior of mypy (specifically that it imports `github` here despite the false `if` condition), can you add a comment to explain what's going on?

Reply:
sure. I have added a comment, PTAL.